runtime/oti/vm_api.h

```diff
@@ -1738,6 +1738,8 @@ setExceptionForErroredRomClass( J9ROMClass *romClass, J9VMThread *vmThread );
 
 /* ---------------- KeyHashTable.c ---------------- */
 
+#define HASH_TABLE_STATE_ITER_FLAG_SKIP_HIDDEN 1
```

Comment:
Please document the purpose of the flag, and I think it would make sense for the constant to be prefixed by `J9_`

Reply:
Added documentation and `J9_` prefix. Also remove `ITER_` from the name.